.env

```diff
@@ -46,3 +46,4 @@ REACT_APP_USER_SERVICE_CURRENT=/api/v1/users/current/
 REACT_APP_USER_SERVICE_LOGOUT=/api/v1/users/logout/
 
 REACT_APP_STATUS_SERVICE=/api/v1/status/
+REACT_APP_AUTH_TOKEN_SERVICE=/api/v1/token/
```

Comment:
As far as I can tell, all these variables are listed in `CONTRIBUTING.md`, so I guess this new one should be added there, too.

Reply:
Thanks for pointing that out. Just updated `CONTRIBUTING.md` with this info.